app/jobs/clean_job.rb

```diff
@@ -0,0 +1,9 @@
+class CleanupWorker < ActiveJob::Base

```

Comment:
Try to have the class name and file name match. Consider renaming this to `DeleteAllCataloguesJob` to accurately describe what it does.